dpnp/dpnp_array.py

```diff
@@ -292,8 +292,10 @@ def __str__(self):
 
         """
 
-        return str(self.asnumpy())
+        return self._array_obj.__str__()
 
+    def __repr__(self):
+        return self._array_obj.__repr__()
```

Comment:
need addition to replace `usm_ndarray` in dpnp output here to something like `dpnp_array`